src/index.html

```diff
@@ -1,14 +1,53 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
-  </head>
-  <body>
+</head>
+<body>
+
     
-    <script src="../dist/index.bundle.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe" crossorigin="anonymous"></script>
-  </body>
+    <div class="container">
+    <h1 class="lead">Leaderboard</h1>    
+    <div class="grid"> 
+        <div class="grid-1">     
+            <div class="flex">
+                <div>
+                    <h1 class="recent">Recent scores</h1>
+                </div>
+                <div>    
+                    <button type="button" class="refresh">Refresh</button>
+                </div>
+            </div>
+                <div id="scores">
+                    <ul class="list-group">
+
+                    </ul>                    
+                </div>
+        </div>        
+
+        <div class="grid-2">
+            <h1 class="add">Add your score</h1>
+            <form>
+                <div >                      
+                  <input type="text" id="name" placeholder="Your name" autocomplete="names">                      
+                </div>
+                <div>                      
+                  <input type="text" id="score"  placeholder="Your score" autocomplete="off">
+                </div>
+               <div class="subm">
+                    <button type="submit" class="sub">Submit</button>
+                </div>
+              </form>
```

Comment:
- [ ] It would be better if users can submit their name and score to the API and see it on the browser, however, we are unable to submit our data to the API . Here is a screenshot of the error in the browser
![Screenshot from 2023-05-25 23-05-22](https://github.com/uno36/Leaderboard/assets/46388113/5f85d14a-dd14-49cb-bd0c-e5d3a69447ab)
Also, because we have not installed and configured `webpack-dev-server` we are unable to run the app on the webpack server as expected. Here is a screenshot of the terminal
![Screenshot from 2023-05-25 23-06-04](https://github.com/uno36/Leaderboard/assets/46388113/494f4e77-2734-4f7c-ac96-83f695213075)